pkg/pillar/dpcmanager/dns.go

```diff
@@ -61,15 +61,11 @@ func (m *DpcManager) updateDNS() {
 		// Set fields from the config...
 		m.deviceNetStatus.Ports[ix].Dhcp = port.Dhcp
 		m.deviceNetStatus.Ports[ix].Type = port.Type
-		_, subnet, _ := net.ParseCIDR(port.AddrSubnet)
-		if subnet != nil {
-			m.deviceNetStatus.Ports[ix].ConfiguredSubnet = subnet
-		}
-		// Start with any statically assigned values; update below
-		m.deviceNetStatus.Ports[ix].DomainName = port.DomainName
-		m.deviceNetStatus.Ports[ix].DNSServers = port.DNSServers
-		m.deviceNetStatus.Ports[ix].ConfiguredNtpServers = port.NTPServers
-		m.deviceNetStatus.Ports[ix].IgnoreDhcpNtpServers = port.IgnoreDhcpNtpServers
```

Comment:
Where are we copying these now?

Reply:
I moved this down to `getDHCPInfo` and `getDNSInfo`, where we are also handling the "ignore-DHCP" options.